src/sentry/search/events/fields.py

```diff
@@ -1312,114 +1335,80 @@ def is_accessible(self, acl=None):
+                    )
+                """.replace(
+                    "\n", ""
+                ).replace(
+                    " ", ""
+                ),
+            ],
             default_result_type="number",
         ),
         Function(
             "count_miserable",
-            required_args=[CountColumn("column"), NumberRange("satisfaction", 0, None)],
+            required_args=[CountColumn("column")],
+            optional_args=[with_default(0, NumberRange("satisfaction", 0, None))],
             calculated_args=[{"name": "tolerated", "fn": lambda args: args["satisfaction"] * 4.0}],
-            aggregate=[
-                "uniqIf",
-                [ArgValue("column"), ["greater", ["transaction.duration", ArgValue("tolerated")]]],
-                None,
-            ],
-            default_result_type="number",
-        ),
-        Function(
-            "count_miserable_new",
-            required_args=[
-                CountColumn("column"),
-            ],
-            aggregate=[
-                "uniqIf",
-                [
-                    ArgValue("column"),
-                    [
-                        "greater",
-                        [
-                            [
-                                "multiIf",
-                                [
-                                    [
-                                        "equals",
-                                        [
-                                            [
-                                                "tupleElement",
-                                                [
-                                                    "project_threshold_config",
-                                                    1,
-                                                ],
-                                            ],
-                                            "'lcp'",
-                                        ],
-                                    ],
-                                    "measurements[lcp]",
-                                    "duration",
-                                ],
-                            ],
-                            [
-                                "multiply",
-                                [
-                                    [
-                                        "tupleElement",
-                                        [
-                                            "project_threshold_config",
-                                            2,
-                                        ],
-                                    ],
-                                    4,
-                                ],
-                            ],
-                        ],
-                    ],
-                ],
-                None,
+            conditional=[
+                ArgValue("satisfaction"),
+                "uniqIf(user, greater(duration, {tolerated:g}))",
```

Comment:
Does this need to be a string? Or should it use the JSON syntax?

Reply:
This whole "[conditional](https://github.com/getsentry/sentry/pull/26429/files#diff-e7d5c143e6b668a98f4dc22ec4fdc3796c413d23cfbd4d3d43c1cd001465cddeR628-R641)" thing is a bit hacky right now. I think it's ok just to support the format string/transform use case. I don't want to also support JSON syntax. We can standardize it in SnQL?

Reply:
I don't want to also add support for JSON syntax here, I just want to keep [this](https://github.com/getsentry/sentry/pull/26429/files#diff-e7d5c143e6b668a98f4dc22ec4fdc3796c413d23cfbd4d3d43c1cd001465cddeR628-R641) simple for now and not have to detect if it's a format string or JSON.

Reply:
Ugh okay, I feel kinda dirty changing all of these to the string form in the tests. Maybe I should introduce two types of conditional statements `conditional_aggregate` and `conditional_transform`? Or should I just rely on type to determine what to do?

Reply:
I'm not against keeping this simple and pushing for the migration to SnQL 👍 as for renaming this to `conditional_*`, I think it may be worth considering to make the intent clearer when reading this.

Reply:
Renamed to `conditional_transform`